README.md

```diff
@@ -284,21 +277,31 @@ will return:
         "aim": {aim},
         "expectationsForMembers": [],
         "members": [
-        {
-            "id": {id},
-            "invitationState": "member"
-        }
+            {
+                "id": {id},
```

Comment:
Have we simply dropped using emails for now? I think we should add a TODO about it, then :-) (specifically that we should solve the problem of inviting people by email)